subsys/net/lib/nrf_cloud/src/nrf_cloud.c

```diff
@@ -259,21 +259,6 @@ int nrf_cloud_shadow_update(const struct nrf_cloud_sensor_data *param)
 	return err;
 }
 
-int nrf_cloud_sensor_attach(const struct nrf_cloud_sa_param *param)
```

Comment:
Have you tried building `samples/nrf9160/lte_ble_gateway`?  It appears to call `nrf_cloud_sensor_attach()` in `main.c`.

Reply:
I meant to remove that 🤦🏻‍♂️

Reply:
removed